build_depends.humble.repos

```diff
@@ -1,35 +1,31 @@
 repositories:
   # core
-  core/autoware_msgs:
-    type: git
-    url: https://github.com/tier4/autoware_auto_msgs.git # TODO(Tier IV): Move to autowarefoundation/autoware_msgs
-    version: tier4/main
-  core/common:
+  core/autoware_common:
     type: git
     url: https://github.com/autowarefoundation/autoware_common.git
     version: main
-  core/autoware:
+  core/autoware.core:
     type: git
     url: https://github.com/autowarefoundation/autoware.core.git
     version: main
+  core/external/autoware_auto_msgs:
+    type: git
+    url: https://github.com/tier4/autoware_auto_msgs.git
+    version: tier4/main
   # universe
-  universe/tier4_autoware_msgs:
+  universe/external/tier4_autoware_msgs:
     type: git
-    url: https://github.com/tier4/AutowareArchitectureProposal_msgs.git # TODO(Tier IV): Rename to tier4/tier4_autoware_msgs
+    url: https://github.com/tier4/tier4_autoware_msgs.git
     version: tier4/universe
-  universe/vendor/grid_map:
+  universe/external/grid_map: # TODO: Use upstream after https://github.com/ANYbotics/grid_map/pull/349 is merged
     type: git
     url: https://github.com/tier4/grid_map.git
```

Comment:
Now, you can use the upstream repository because https://github.com/ANYbotics/grid_map/pull/349 has been merged.

Reply:
Ah, exactly!

Reply:
89765a1f9143ef1ccf062f7d408d94884596d28e
d7efaf6c593a4edaca83fc1091911ebbd2bc067f